script/functions.sh

```diff
@@ -11,6 +11,10 @@ export JRUBY_OPTS=${JRUBY_OPTS:-"--server -Xcompile.invokedynamic=false"}
 SPECS_HAVE_RUN_FILE=specs.out
 MAINTENANCE_BRANCH=`cat maintenance-branch`
 
+# Don't allow rubygems to pollute what's loaded. Also, things boot
+# faster without the extra load time of rubygems.
+export RUBYOPT="--disable=gem"
```

Comment:
It looks like this needs to be made conditional, so that it only runs on MRI 1.9+.  MRI 1.8.7 and JRuby fail with this option as it is unrecognized.